cmd/background-heal-ops.go

```diff
@@ -50,6 +50,15 @@ type healRoutine struct {
 
 // Add a new task in the tasks queue
 func (h *healRoutine) queueHealTask(task healTask) {
+	opts := globalHealConfig
+
+	// Wait and proceed if there are active requests
+	waitForLowHTTPReq(int32(globalEndpoints.NEndpoints()), opts.Sleep)
+
+	if opts.Bitrot {
+		task.opts.ScanMode = madmin.HealDeepScan
```

Comment:
Wont this also override any values sent from admin API?

Reply:
This part is only used by internal routines @klauspost it is not invoked through admin API.